apps/nrm/models/submission.js

```diff
@@ -0,0 +1,183 @@
+
+  response.ReportedCase = data['reported-to-police'] === 'yes' ? 'Yes' : 'No';
+  response['CaseContact.Police_force'] = data['reported-to-police-police-forces'];
+  response['CaseContact.Alternate_contact'] = data['fr-alternative-contact'];
+  response['CaseContact.Alternate_first_responder'] = data['fr-alternative-contact'];
+  response['CaseContact.Local_authority'] = data['local-authority-contacted-about-child-local-authority-name'];
+
+  response['CaseContactOrganisation.Police_force'] = data['reported-to-police-police-forces'];
+  response['CaseContactOrganisation.Local_authority']
+    = data['local-authority-contacted-about-child-local-authority-name'];
+
+  response['CaseContactEmail.Police_force'] = data['local-authority-contacted-about-child-local-authority-email'];
+  response['CaseContactEmail.Alternate_contact'] = data['fr-alternative-contact'];
+  response['CaseContactEmail.Alternate_first_responder'] = data['fr-alternative-contact'];
+  response['CaseContactEmail.Local_authority'] = data['local-authority-contacted-about-child-local-authority-email'];
+
+  // response['CaseContactCustom1.Police_force'];
+  // response['CaseContactFirstName.Alternate_contact'];
+  // response['CaseContactFirstName.Local_authority'];
+  // response['CaseContactAddress.Alternate_contact'];
+  // response['CaseContactTown.Alternate_contact'];
+  // response['CaseContactCounty.Alternate_contact'];
+  // response['CaseContactPostcode.Alternate_contact'];
+  // response['CaseContactCustom1.Alternate_contact'];
+
+  response['CaseContactPhone.Local_authority'] = data['fr-details-phone'];
+
+  response.SafeToEmail = data['pv-contact-details-email-check'] === 'true' ? 'Yes' : 'No';
+  response.SafeToPostal = data['pv-contact-details-post-check'] === 'true' ? 'Yes' : 'No';
+
+  response.CountryLabel = data['fr-location'];
+  // response.Country = data['fr-location'] //eg ENG;
+
+  response.HowToNotfy = data['pv-contact-details'] === 'email' ? 'Email' : 'Post';
+  response.CanPoliceContactPV = data['co-operate-with-police'] === 'yes' ? 'Yes' : 'No';
+  response.CIDReference = data['reported-to-police-crime-reference'];
+  response.HORefType = data['pv-ho-reference-type'];
+  response.NRMOrDuty = data['pv-want-to-submit-nrm'] === 'yes' ? 'Yes' : 'No';
+  // response.DTNReason;
+  response.NeedSupport = data['does-pv-need-support'] === 'yes' ? 'Yes' : 'No';
+  response.SendDecisionTo = data['who-contact'] === 'potential-victim' ? 'PV' : 'FR';
+
+  response['Agent.Forename1'] = data['fr-details-first-name'];
+  response['Agent.Name'] = data['fr-details-last-name'];
+  response['Agent.Email'] = data['user-email'];
+  response['Agent.Phone'] = data['fr-details-phone'];
+  response['Agent.Jobtitle'] = data['fr-details-role'];
+  response['Agent.Organisation'] = data['user-organisation'];
+  // response['Agent.Address'];
+  // response['Agent.Town'];
+  // response['Agent.County'];
+  // response['Agent.Postcode'];
+  // response['Agent.Mobile'];
```

Comment:
comment?

Reply:
yes because it's in the API but we don't know what to do with it yet